gs_examples/sci_cam_image2/app/src/main/java/gov/nasa/arc/irg/astrobee/sci_cam_image2/SciCamImage2.java

```diff
@@ -105,7 +105,7 @@ protected void onCreate(Bundle savedInstanceState) {
         Intent in = getIntent();
         Bundle b = in.getExtras();
         if (b != null) {
-            dataPath = (String)b.get("data_path");
+            dataPath = (String)b.get("path") + File.separator + "delayed";
```

Comment:
Actually it should be "data_path" because it is myself who sets this in the caller, so in StartSciCamImage.java, so here, in the callee, in SciCamImage.java, the same name should be used. These are independent of the GS manager code, all the action happens in my own code.